crm/optimize/outlook_extension.rst

```diff
@@ -137,7 +137,27 @@ Connect to your database
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+The extension is **free** and doesn't require any purchase to be installed and used.
+
+However, this extension provides you with *Lead Enrichment* in the **Company Insights** section.
+This service is part of a paid service known as **Lead Generation**.
+
+The extension allows you to test this service for free, whether you connect the extension to a
+database or not. After a while, the extension asks you to buy credits to keep using this additional
+service.
+
+.. image:: media/outlook-extension-credits-message.png
+   :align: center
+   :alt: "Could not auto-complete the company: not enough credits!" warning message in the Outlook
+         extension
+
+You can buy more *Lead Generation* credits to keep using this service or disregard this message and
+keep using the extension for free without *Lead Enrichment*.
```

Comment:
I think it could be inside of a "tip".

Reply:
![image](https://user-images.githubusercontent.com/46810843/100073932-67d82280-2e3e-11eb-850b-bcf941686b41.png)